thirdparty/openapi3/schemas/ThirdpartyRequestsTransactionsIDPatchResponse.yaml

```diff
@@ -2,13 +2,13 @@ title: ThirdpartyRequestsTransactionsIDPatchResponse
 type: object
 description: 'The object sent in the PATCH /thirdpartyRequests/transactions/{ID} callback.'
 properties:
-  fulfilment:
-    $ref: '../../../v1.0/openapi3/schemas/IlpFulfilment.yaml'
-  completedTimestamp:
-    $ref: '../../../v1.0/openapi3/schemas/DateTime.yaml'
-  transferState:
+  transactionId:
+    $ref: '../../../v1.0/openapi3/schemas/CorrelationId.yaml'
+  transactionRequestState:
+    $ref: '../../../v1.0/openapi3/schemas/TransactionRequestState.yaml'
+  transactionState:
     $ref: '../../../v1.0/openapi3/schemas/TransferState.yaml'
```

Comment:
maybe this should be `transactionState`

Reply:
title: TransactionState
type: string
enum:
  - RECEIVED
  - PENDING
  - COMPLETED
  - REJECTED

`transactionState` has no COMMITED enum 
```
**""PATCH /thirdpartyRequests/transactions/123""**
""FSPIOP-Source: switch""
""FSPIOP-Destination: pispa""
{
  "transactionId": "987",
  "transactionRequestState": "ACCEPTED",
  "transactionState": "COMMITTED"
}
```

Where as `transferState` does
```
title: TransferState
type: string
enum:
  - RECEIVED
  - RESERVED
  - COMMITTED
  - ABORTED
```

This where my confusion lies. Should I make a new ThirdpartyTransactionState to have COMMITTED? or is COMMITTED the wrong enum in the sequence diagrams. @lewisdaly

Reply:
Ah I see... I think that means the sequence diagrams are incorrect then. We've been so worried about designing the method for this call that we haven't spent enough time going over it's contents.

Reply:
Ok. Updated it to reference `TransactionState`